app/models/form_profile.rb

```diff
@@ -276,7 +277,7 @@ def vet360_contact_info
     return @vet360_contact_info if @vet360_contact_info_retrieved
 
     @vet360_contact_info_retrieved = true
-    if Settings.vet360.prefill && user.vet360_id.present?
+    if VAProfile::Configuration::SETTINGS.prefill && user.vet360_id.present?
```

Comment:
It looks like prefill is an option in some of the other entries in settings.yml but it doesn't look like it is part of vet360/va_profile. Granted, this isn't a change that was made in this PR but it might be worth looking at.

Reply:
looks like it's set in the devops settings.yml files